app/views/recipes/index.html.haml

```diff
@@ -1,26 +1,19 @@
 .page-header
-  %h1.text-uppercase Recipes
+  %h1.text-uppercase.text-center My Recipes
 
-%table.table.table-condensed.table-responsive
-  %thead
-    %tr
-      %th Name
-      %th Preparation
-      %th Time
-      %th Cost
-      %th
-  %tbody
-    - @recipes.each do |recipe|
-      %tr
-        %td= link_to recipe.name, recipe
-        %td= recipe.preparation
-        %td= recipe.preparation_time
-        %td= number_to_currency recipe.cost
-        %td.text-right
-          = link_to 'Edit', edit_recipe_path(recipe), class: 'btn btn-sm btn-primary'
-          = link_to 'Destroy', recipe, method: :delete, data: { confirm: 'Are you sure?' }, class: 'btn btn-sm btn-danger'
-%br
-%br
 .row
-  .col-md-4.col-md-offset-4
-    = link_to 'New Recipe', new_recipe_path, class: 'btn btn-primary btn-block btn-lg'
+  - @recipes.each_slice(3) do |recipe_two|
+    -recipe_two.each do |recipe|
+      .col-md-4.col-md-4.col-md-4.extramargin
+        .container-box.center-block
+          %div.img-rounded.center-block{style:"width:250px; height:200px; border:1px solid #000; margin-bottom:15px; background-color:#eee"}
```

Comment:
Since, we haven't installed the carrier waver gem, i just us that div so that it is the place holder for the image, Just to see the difference in layout